spec/dummy/db/schema.rb

```diff
@@ -213,6 +213,13 @@
     t.string "color"
   end
 
+  create_table "types", force: :cascade do |t|
```

Comment:
This probably does not belong here. Could you please remove it?

Reply:
@adrianthedev sure, I'll remove this changes. Thanks